### PR TITLE
Https2

### PR DESCRIPTION
**Pull request by jdoconnor**
@lilianchen1 and I got this to work
@alan-andrade @lumoslabs/rails -- please review. First commit is a rebase to what is on production. after that is the commit that makes it work

### CODE REVIEW

**Comment by jdoconnor** [comment]
@alan-andrade can you give me (or a lumos user) permissions to update this? (or do you want to do it) -- on google's extension store.

**Comment by ababx** [comment]
background.css

``` css
@font-face {
  font-family: 'Museo Sans';
  src: url(MuseoSans.woff);
}

a {
  text-decoration: none;
  color: #aeaeae;
}

body{
  background-color: #f6f7f8;
}

h3{
  color: #7F7F7F;
  font-family: "Museo Sans","Lucida Grande",Arial,sans-serif;
  font-weight: bold;
}

h4{
  color: #aeaeae;
  font-family: "Museo Sans","Lucida Grande",Arial,sans-serif;
  font-size: 0.9em;
  font-weight: normal;
}

.content{
  margin-left: auto;
  margin-right: auto;
  width: 680px;
  margin-top: 30px;
}

#game-container{
  background-color: white;
  border: solid 1px #e2e2e2;
  box-shadow: 0px 1px 2px #e2e2e2;
  width:100%;
}

#game-canvas{
  margin: 20px 20px 10px 20px;
}

#outerdiv {
width:684px;
height:554px;
overflow:hidden;
position:relative;
}
#innerdiv {
position:absolute;
top:-129px;
left:-146px;
width:1280px;
height:1200px;
}
```

**Comment by ababx** [comment]
popup.css

``` css
body{
  font-family: 'Museo Sans', 'Lucida Grande', Arial, sans-serif;
  padding: 0;
  margin:0;
  color: #000 !important;
}

#wrapper {
  width: 350px;
  padding: 0;
  margin:0;
}

#error {
    display: none;
    color: #be1e2d;
    padding-bottom: 20px;
}

h3{
  color: #000;
  font-size: 1em;
  font-family: "Museo Sans","Lucida Grande",Arial,sans-serif;
  font-weight: bold;
}

hr{
  border-top: 1px solid #ccc;
}
.rich-input{
  max-width: 310px;
  font-family: "Museo Sans","Lucida Grande",Arial,sans-serif;
  font-size: 0.9em;
  color: #333;
}

.rich-input input{
  font-family: "Museo Sans","Lucida Grande",Arial,sans-serif;
  width: 40%;
  height: 45px;
  box-sizing: border-box;
  font-size: 1.1em;
  margin-bottom: 4px;
  margin-right:10px;
  vertical-align: middle;
  padding-left: 10px;
}
 .schedule{
  padding:0 20px;
  border-bottom: 1px solid #ccc;
}
 .games{
  padding:0 20px;
  margin-bottom: 20px;
}
.btn{
  font-family: "Museo Sans","Lucida Grande",Arial,sans-serif;
  width: 50%;
  color: #fff;
  border-bottom-color: #c94727;
  letter-spacing: 0px;
  font-size: 1.1em;
  cursor: pointer;
  -webkit-appearance: button;
  text-shadow: none;
  border-width: 0 0 2px 0;
  -webkit-box-shadow: inset 0px 0px 0px 0px;
  -moz-box-shadow: inset 0px 0px 0px 0px;
  box-shadow: inset 0px 0px 0px 0px;
  padding: 14px 30px 12px 30px;
  font-weight: 500;
  background-color: #f1693c;
  background-repeat: repeat-x;
  border-color: rgba(0,0,0,0.1) rgba(0,0,0,0.1) rgba(0,0,0,0.25);
  display: inline-block;
  -webkit-border-radius: 4px;
  -moz-border-radius: 4px;
  border-radius: 4px;
  margin: 0 0 20px 0;
  box-sizing: border-box;
  text-align: center;
}

.ui-state-default{
color: #1c94c4 !important;
}

.ui-button{
  font-family: "Museo Sans","Lucida Grande",Arial,sans-serif !important;
  font-size: .8em !important;
  text-align: left !important;
  background: #efefef !important;
  border: 1px solid #aaa !important;
  margin: 2px !important;
}


.ui-state-active{
  color: #fff !important;
}
.ui-state-active, .ui-widget-content .ui-state-active, .ui-widget-header .ui-state-active {
  border:1px solid #6BBD44 !important;
  background: #6BBD44 !important;
}

.ui-state-hover{
  border:1px solid #3F803C !important;
}
```

**Comment by alan-andrade** [comment]
@jdoconnor  @lilianchen1 I added the CSS that Abhi had on his computer.

Seems like we're good to go after we solve the `SAMEORIGIN` thing. @jdoconnor Ideas on how do we solve that ?

File: js/background.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
+// hijack web requests made by chrome to remove security headers
+chrome.webRequest.onHeadersReceived.addListener(
```
Comment (jdoconnor) [inline js/background.js:2]: @alan-andrade this is the sameorigin fix.

**Comment by jdoconnor** [comment]
i still don't appear to have access to upload. Can one of you?

**Pull request merged**